pkg/deployment/deployment.go

```diff
@@ -326,6 +327,35 @@ func addServiceExtraMounts(
 
 		aeeSpec.ExtraMounts = append(aeeSpec.ExtraMounts, volMounts)
 	}
+
+	// Add mount for TLS certs
+	if service.Spec.HasTLSCerts {
+		volMounts := storage.VolMounts{}
+		secretName := nodeSet.Name + "-" + service.Name + "-certs"
```

Comment:
Lets move this name generation into a function so it is kept in sync between the place we create the secret and the place we use the secret

Reply:
Ack, good idea.